engine/src/main/java/io/zeebe/engine/processing/streamprocessor/MigratedStreamProcessors.java

```diff
@@ -63,7 +63,11 @@
 
     MIGRATED_VALUE_TYPES.put(
         ValueType.MESSAGE_SUBSCRIPTION,
-        record -> record.getIntent() == MessageSubscriptionIntent.CORRELATING);
+        MIGRATED_INTENT_FILTER_FACTORY.apply(
+            List.of(
+                MessageSubscriptionIntent.CREATE,
+                MessageSubscriptionIntent.CREATED,
```

Comment:
CREATED?

Reply:
Yes. We list commands and events. If it is in the list then the stream processor replays the event.